command/util.go

```diff
@@ -0,0 +1,75 @@
+package command
```

Comment:
not sure about the file name since all functions below are used in versioning. maybe it might be a part of `bucket-version.go`.

Reply:
I will rename it to `validation.go`.